### PR TITLE
Add executor option to blackhole traffic to private ranges.

### PR DESCRIPTION
**Pull request by vadimberezniker**
**Related issues**: N/A

### CODE REVIEW

File: enterprise/server/remote_execution/containers/podman/podman.go [hunk 1]
```diff
@@ -874,6 +867,25 @@ func ConfigureSecondaryNetwork(ctx context.Context) error {
 	if err := networking.AddRouteIfNotPresent(ctx, route); err != nil {
 		return err
 	}
+
+	return nil
+}
+
+// Configure the secondary network for podman so that traffic from podman will be routed through
```
Comment (luluz66) [inline enterprise/server/remote_execution/containers/podman/podman.go:874]: nit: ConfigureIsolation configures...also maybe mention the blackholing

**Review by tylerwilliams — APPROVED** [review]

File: server/util/networking/networking.go [hunk 2]
```diff
@@ -496,7 +515,32 @@ func routingTableContainsTable(tableEntry string) (bool, error) {
 	return false, nil
 }
 
-// ConfigurePolicyBasedRoutingForNetworkWIthRoutePrefix configures policy routing for secondary
+// ConfigureRoutingForIsolation sets up a routing table for handling network
+// isolation via either a secondary network interface or blackholing.
+func ConfigureRoutingForIsolation(ctx context.Context) error {
+	if !IsSecondaryNetworkEnabled() && !*blackholePrivateRanges {
```
Comment (luluz66) [inline server/util/networking/networking.go:521]: nit: use IsPrivateRangeBlackholingEnabled()

File: enterprise/server/remote_execution/containers/podman/podman.go (hunk 1 above)
Comment (vadimberezniker) [reply enterprise/server/remote_execution/containers/podman/podman.go]: done

File: server/util/networking/networking.go (hunk 2 above)
Comment (vadimberezniker) [reply server/util/networking/networking.go]: done

**Pull request merged**